eng/pipelines/docindex.yml

```diff
@@ -49,13 +49,30 @@ jobs:
       - template: /eng/common/pipelines/templates/steps/set-daily-docs-branch-name.yml
         parameters:
           DailyBranchVariableName: DailyDocsBranchName
+      # Sync docs fork repo onboarding files/folders
+      - pwsh: |
+          $dailyDocsRepoLocation = "$(DocRepoLocation)/dailyDocs"
```

Comment:
Do we need to set this dynamically or can we set it statically where we define DocRepoLocation?

Reply:
`DocRepoLocation` means Azure/main -> Azure/main
`dailyDocsRepoLocation` mean azure-sdk/daily -> Azure/daily

(-> means finally push to)

Reply:
These two steps both happen in schedule run

Reply:
I think we can eliminate this `pwsh` task by setting the variable above: 

```
    variables:
      DocRepoLocation: $(Pipeline.Workspace)/docs
      DailyDocRepoLocation: $(Pipeline.Workspace)/dailyDocsFork  # Add this line
      DocRepoOwner: Azure
      DocRepoName: azure-docs-sdk-java
```

That would eliminate the need to create this variable at runtime and the [sparse checkout process handles creating the directory](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/pipelines/templates/steps/sparse-checkout.yml#L32).

Reply:
Yes that is wat I was wondering about. However see my other comment as well as we should consider just adding a remote to the fork instead of a separate clone.